tools/k8s-stack/pkg/stack.go

```
@@ -0,0 +1,110 @@
+	}
+
+	resourceOptions = append(resourceOptions, pulumi.DependsOn([]pulumi.Resource{namespace}))
+
+	cmp.Postgres, err = NewPostgresComponent(ctx, "postgres", &PostgresComponentArgs{
+		Namespace: namespaceName,
+	}, resourceOptions...)
+	if err != nil {
+		return nil, fmt.Errorf("creating Postgres component: %w", err)
+	}
+
+	cmp.Ledger, err = pulumi_ledger.NewComponent(ctx, "ledger", &pulumi_ledger.ComponentArgs{
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)